app/src/main/java/app/grapheneos/camera/ui/activities/MoreSettings.kt

```diff
@@ -96,6 +96,14 @@ open class MoreSettings : AppCompatActivity(), TextView.OnEditorActionListener {
                 sIAPToggle.isChecked
         }
 
+        val sIVPToggle = binding.saveVideoAsPreviewToggle
+
+        sIVPToggle.isChecked = camConfig.saveVideoAsPreviewed
+
+        sIVPToggle.setOnClickListener {
```

Comment:
Tapping on "Save video as previewed" settings item doesn't toggle the setting, only tapping on the small toggle widget on the right does. It's inconsistent with the rest of settings.

Reply:
I have fixed this in the recently pushed set of commits to the branch

Reply:
I have fixed this in the recently pushed set of commits to the branch